Symfony/CS/Fixer/Contrib/PhpdocSingleLineVarSpacingFixer.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Fixer for part of rule defined in PSR5 ¶7.22.
+ *
+ * @author SpacePossum
+ */
+final class PhpdocSingleLineVarSpacingFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        /** @var Token $token */
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $token) {
+            $content = $token->getContent();
+            $content = preg_replace_callback(
+'               #/\*\*[ \t]*@var[ \t]+([\S]+)[ \t]+(\$[\S]+)?[ \t]+([^\n]*)\*/#',
```

Comment:
> would it process faster?

I think so, because there are less possibilities that have to be checked by the regex engine.
It is faster for non-matching doc blocks, not for the matching ones.

> Both strings cannot occur more than once

The first (`/**`) can occur multiple times..